### PR TITLE
Add predict model, random_search_cv_scores.csv, environment file

### PR DESCRIPTION
**Pull request by karanpreetkaur**
Added random_search_cv_scores.csv which contains scores of cross validation folds in randomized search.

### CODE REVIEW

**Review by Arushi282 — APPROVED** [review]

**Comment by Arushi282** [comment]
Okay I actually need to clarify something, in your random search code, you test for these values of C: [0.001, 0.01, 0.1, 1, 10, 100, 1000], but in the random_scores_cv data set, the splits are only seen for values up until 100, it that because random search does not necessarily test for all values of C provided?

**Comment by karanpreetkaur** [comment]
@Arushi282 Yes, I think randomized search doesn't go over each value in the range plus I also set the max_iter i.e. maximum iterations so it's possible that iterations were exhausted by C = 100. Hope this clarifies.

**Review by thayeylolu — APPROVED** [review]

File: src/credit_default_predict_model.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import RandomizedSearchCV
+
+opt = docopt(__doc__)
+
+def random_search_cv_score(model_tuned, out_dir):
+  """
+  Returns the f1 score for each cross validation folds in RandomizedSearch
```
[the remark this answers is not in the document]
Comment (karanpreetkaur) [reply src/credit_default_predict_model.py]: @liannah can you review docstrings now ?

[the remark this answers is not in the document]
Comment (liannah) [reply src/credit_default_predict_model.py]: No comments. Looks good, would not make sense to have an example here, so I think docstring are ready. @karanpreetkaur

**Comment by liannah** [comment]
Please let me know, if it is ready for the merge. I think @Arushi282 needs to confirm, as this merge can cause conflicts for her PR. @karanpreetkaur if she is okay, I can merge it as well.

File: src/credit_default_predict_model.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                                     scoring=scoring, 
+                                     refit='f1', 
+                                     random_state=123)
+  random_search.fit(X_train, y_train)
+  # Test to check the return type of tuned model
+  assert isinstance(random_search, RandomizedSearchCV), "random_search is not a RandomizedSearchCV model"
+  return random_search
+
+def main(train_path, test_path, out_dir):
+  # Read train and test data from respective arguments in opt
```
Comment (thayeylolu) [inline src/credit_default_predict_model.py:115]: i think this file is a little over commented

**Review by thayeylolu — CHANGES_REQUESTED** [review]

File: src/credit_default_predict_model.py (hunk 2 above)
Comment (liannah) [reply src/credit_default_predict_model.py:115]: I think it is good actually

**Pull request merged**